packages/material-ui-lab/src/Pagination/Pagination.js

```diff
@@ -7,48 +7,63 @@ import PaginationItem from '../PaginationItem';
 
 export const styles = {
   /* Styles applied to the root element. */
-  root: {
+  root: {},
+  /* Styles applied to the ul element. */
+  ul: {
     display: 'flex',
     flexWrap: 'wrap',
     alignItems: 'center',
+    padding: 0,
+    margin: 0,
     listStyle: 'none',
-    padding: 0, // Reset
-    margin: 0, // Reset
   },
 };
 
+function defaultGetAriaLabel(type, page, selected) {
+  if (type === 'page') {
+    return `${selected ? '' : 'Go to '}page ${page}`;
+  }
+  return `Go to ${type} page`;
+}
+
 const Pagination = React.forwardRef(function Pagination(props, ref) {
   const {
     children,
     classes,
     className,
     color = 'standard',
-    getItemAriaLabel: getAriaLabel,
+    getItemAriaLabel: getAriaLabel = defaultGetAriaLabel,
     items,
     renderItem = item => <PaginationItem {...item} />,
     shape = 'round',
-    size,
+    size = 'medium',
     variant = 'text',
     ...other
   } = usePagination({ ...props, componentName: 'Pagination' });
 
-  const itemProps = { color, getAriaLabel, shape, size, variant };
-
   return (
-    <ul
-      role="navigation"
+    <nav
       aria-label="pagination navigation"
       className={clsx(classes.root, className)}
       ref={ref}
       {...other}
     >
-      {children ||
-        items.map(item => (
-          <li key={item.type !== undefined ? item.type : item.page.toString()}>
-            {renderItem({ ...item, ...itemProps })}
-          </li>
-        ))}
-    </ul>
+      <ul className={classes.ul}>
```

Comment:
```suggestion
      <ul role="list" className={classes.ul}>
```
?

Reply:
It seems that the `<ul>` element has a default role of "list".

Reply:
https://github.com/mui-org/material-ui/pull/19612#issuecomment-583770634

Reply:
Default roles should never be used. 

> [...] default implicit ARIA semantics is unnecessary and not recommended as these properties are already set by the browser.

-- https://www.w3.org/TR/html52/dom.html#do-not-set

I would prefer that over a blog post (even if it comes from an editor). Unless there's some issue on the spec concerning this.

Reply:
> Default roles should never be used.

That isn't what the sentence you quoted quote says ("not recommended").

> I would prefer that over a blog post

Blog post? (https://w3c.github.io/html-aria/#el-ul)

> Unless there's some issue on the spec concerning this.

Strict conformance to the spec (which is actually only a recommendation in this case) over pragmatic choices that help make content more accessible seems like the wrong tradeoff to me.

Reply:
I don't think that we should set the default role, it's redundant.

Reply:
But I'm wondering, what does this imply?

> Note: some user agents suppress a list's implicit ARIA semantics if list markers are removed. Authors can use role=list to reinstate the role if necessary.

Reply:
> Maybe to what we set with CSS list-style: none;?

Correct, although the reference I found that describes this more explicitly now eludes me.

> > Note: some user agents suppress a list's implicit ARIA semantics if list markers are removed. **Authors can use role=list to reinstate the role if necessary.**

> *https://w3c.github.io/html-aria/#el-ul*

That was my point: (https://github.com/mui-org/material-ui/pull/19612#issuecomment-583770634)

Reply:
Ok, great, so we are all on the same page now :).

Hard call to make. I think that without more information, following the "others" (all the first links we have for the query "pagination accessibility" on Google) would be the safest move. It would also match the breadcrumbs.

Reply:
> so we are all on the same page now :)

B'dum tish!

Speaking of the first page of Google, I was curious to see how they mark it up for the "Goooooogle" pagination, but they use a table! 🤣

Given that the majority of pagination implementations that use `<ul>` do not add `role="list"`, even if they perhaps should do, the "principal of least surprise" suggests that we follow suit so that a Safari / Voiceover user's experience is at least consistent with other implementations, even if not altogether correct.

Reply:
I hadn't this principle of "past experience" in mind (that I love BTW). I was proposing to leverage the "experience"/"exposure" of others. I think that the probability of doing it right is in their favor; not ours.

Reply:
> I think that the probability of doing it right is in their favor; not ours.

You mean like Google using a table for pagination? :)

No, the guidance on this is pretty clear cut, it's just that nobody is following it. Following the crowd in this case isn't about others being correct, but about not bucking the trend. Subtly different.

Incidentally, there's no consistency with the use of `<nav>` to wrap the `<ul>`, and for those that do, whether or not to add `role="navigation"`, so we still have that to debate. 😄

Reply:
I can't see clearly through these conflicting signals. Maybe the following would help: What happens if we set a default role? How can it go bad?

Reply:
Worst case:
1. Validator warnings

But see also: https://www.scottohara.me/blog/2019/03/26/a-warning-about-warnings.html

"Sometimes you may come across gaps in the assertions these tools are based on. Or a best practice may dictate that you don’t double up ARIA roles on their native elements, when that’s absolutely what you need to do to ensure cross-browser and assistive technology feature parity."

2. User "surprise" (for Safari with Voiceover users) when the experience is not consistent with other pagination implementations they are used to (that use `<ul>` with `list-style: none` but don add `role="list"`).


For that reason I'm suggesting (last paragraph here: https://github.com/mui-org/material-ui/pull/19612#discussion_r376790283) that we go with the flow rather than "fix" it. If we get feedback that it causes problems, we can revisit it.

Reply:
Ok awesome!